redaxo/src/core/pages/system.settings.php

```diff
@@ -88,6 +88,42 @@
             $success = rex_i18n::msg('info_updated');
         }
     }
+} elseif ('updateeditor' === $func) {
+    $editor = rex_post('editor', [
+        ['name', 'string', null],
+        ['basepath', 'string', null],
+        ['update_cookie', 'bool', false],
+        ['delete_cookie', 'bool', false],
+    ]);
+
+    $cookieOptions = ['samesite' => 'strict'];
+
+    if ($editor['delete_cookie']) {
+        rex_response::clearCookie('editor', $cookieOptions);
+        rex_response::clearCookie('editor_basepath', $cookieOptions);
+        unset($_COOKIE['editor']);
+        unset($_COOKIE['editor_basepath']);
+
+        $success = rex_i18n::msg('system_editor_success_cookie_deleted');
+    } elseif ($editor['update_cookie']) {
+        rex_response::sendCookie('editor', $editor['name'] ?: null, $cookieOptions);
```

Comment:
ist der editor-name and dieser stelle nicht bereits immer null?

Reply:
Nein, aus `<option value="">Deaktiviert</option>` wird ein leerer String.

Reply:
können wir diesen part dann oben beim handling der POST values machen, statt hier zwischen drinn? (Zeile 92)